src/modules/Modules.cpp

```diff
@@ -31,6 +31,7 @@
 #include "modules/ExternalNotificationModule.h"
 #include "modules/RangeTestModule.h"
 #if (defined(ARCH_ESP32) || defined(ARCH_NRF52)) && !defined(CONFIG_IDF_TARGET_ESP32S2)
+#include "modules/DetectionSensorModule.h"
```

Comment:
excluded for RP2040?

Reply:
Moved it up. I didn't even notice it was macro guarded until portduino target failed 😅